src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.cs

```diff
@@ -101,6 +101,13 @@ private void Init(string pattern, RegexOptions options, TimeSpan matchTimeout, C
             roptions = options;
             internalMatchTimeout = matchTimeout;
 
+#if DEBUG
+            if (Debug)
+            {
+                System.Diagnostics.Debug.Write($"Pattern:     {pattern}");
```

Comment:
Why just the pattern?  Why not also options, timeout, culture, etc.?

Reply:
added